<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaleClass</class>
 <widget class="QWidget" name="ScaleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>172</width>
    <height>31</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>172</width>
    <height>31</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>188</width>
    <height>31</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Scale</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pbD0">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>8</y>
     <width>14</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>14</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>14</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DS-Digital</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbD1">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>8</y>
     <width>14</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>14</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>14</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DS-Digital</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbD3">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>2</y>
     <width>20</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DS-Digital</family>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbD2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>8</y>
     <width>14</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>14</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>14</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DS-Digital</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbD4">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>2</y>
     <width>20</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DS-Digital</family>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbD5">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>2</y>
     <width>20</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DS-Digital</family>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbD6">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>2</y>
     <width>20</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DS-Digital</family>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbD7">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>2</y>
     <width>20</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DS-Digital</family>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbD8">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>2</y>
     <width>20</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DS-Digital</family>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbD9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>2</y>
     <width>20</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DS-Digital</family>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDot">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>2</y>
     <width>12</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>12</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>12</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DS-Digital</family>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
